src/bin/vip-import-media.js

```diff
@@ -106,17 +106,30 @@ Are you sure you want to import the contents of the URL?
 		'Check the status of a currently running media import or retrieve an error log of the most recent media import.'
 	)
 	.command( 'abort', 'Abort the media import currently in progress.' )
-	.option( 'exportFileErrorsToJson', 'Format the error log in JSON. Default is TXT.' )
+	.option( 'exportFileErrorsToJson', 'Alias for --export-file-errors-to-json. (deprecated)' )
+	.option( 'export-file-errors-to-json', 'Format the error log in JSON. Default is TXT.' )
```

Comment:
Do we need to explicitly add short form options as you've done with `allow-all-file-types` for these too?